java-squid/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -574,6 +575,17 @@ private static boolean isLocalMethodInvocation(MethodInvocationTree tree) {
     return false;
   }
 
+  public static boolean isObjectsMethod(MethodInvocationTree syntaxNode, String methodName, int numberOfParameter) {
```

Comment:
To me it has nothing to do in the ExplodedGraphWalker and should be moved in the ConstraintManager.